source/WanManager/wanmgr_policy_autowan_impl.c

```diff
@@ -981,17 +1039,14 @@ static WcFmobPolicyState_t Transition_FixedWanInterfaceUp(WanMgr_Policy_Controll
         if(pFixedInterface->VirtIfList->IP.ConnectivityCheckType == WAN_CONNECTIVITY_TYPE_TAD)
         {
             WanMgr_disable_ra(pFixedInterface->VirtIfList->Name);
-            WanMgr_Sysevent_SetConnectivityCheckState(NET_CONNNECTIVITY_CHECK_STARTING);
             pFixedInterface->VirtIfList->IP.Ipv4ConnectivityStatus = WAN_CONNECTIVITY_UP;
             pFixedInterface->VirtIfList->IP.Ipv6ConnectivityStatus = WAN_CONNECTIVITY_UP;
+            WanMgr_Sysevent_SetConnectivityCheckState(true, NET_CONNNECTIVITY_CHECK_STARTING);
         }
     }
 #endif
     // start wan
-    if (WanMgr_GetWanServiceStatus() == 0)
-    {
-        wanmgr_setwanstart();
-    }
+    wanmgr_setwanstart();
```

Comment:
yes it is applicable for both. but there is no previous service wan running in fixed policy before this call , service wan start here at first time from fixed policy. 
BTW, its wan start not wan-restart. so it will never stop any running service wan if there.